CHANGELOG.md

```diff
@@ -7,6 +7,13 @@
 Cromwell now allows user defined retries. With `memory-retry` config you can specify an array of strings which when encountered in the `stderr` 
 file by Cromwell, allows the task to be retried with multiplier factor mentioned in the config. More information [here](https://cromwell.readthedocs.io/en/stable/backends/Google/).
 
+### Papi V2 Localization Using GCR [(#5200)](https://github.com/broadinstitute/cromwell/pull/5200)
+
+The Docker image for the Google Cloud SDK was previously only [published on Docker
+Hub](https://hub.docker.com/r/google/cloud-sdk). Now that the image is [publicly hosted in
+GCR](http://gcr.io/google.com/cloudsdktool/cloud-sdk), Papi V2 jobs will localize inputs and delocalize outputs using
+the GCR image.
```

Comment:
What is the benefit of using the image from GCR v/s Dockerhub?

Reply:
This is a continuation of the Papi networking (VPC / NAT / private IP) work. Not using Docker Hub means *Cromwell* doesn't need any resources outside of Google Cloud, so now a workflow can use things like private IP address and still localize files. Using private IPs allows one to run more machines, as it won't trip over a public ip address quotas-- or even limits in availability. NOTE: NAT was a workaround for keeping public ip address numbers down too, but I don't know the status of Cromwell/Workbench implementing that feature.

I don't know if this 100% fixes networking features for Papi V2 in Cromwell/Workbench, but it's a huge step.

- https://cloud.google.com/genomics/reference/rest/Shared.Types/Metadata#Network.FIELDS.use_private_address
- https://cromwell.readthedocs.io/en/stable/RuntimeAttributes/#noaddress